packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -352,10 +355,30 @@ pub enum RemotePatternProtocal {
 #[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize, TraceRawVcs)]
 #[serde(rename_all = "camelCase")]
 pub struct ExperimentalTurboConfig {
-    pub loaders: Option<IndexMap<String, WebpackLoaderConfigItems>>,
+    /// This option has been replace by `rules`.
+    pub loaders: Option<JsonValue>,
```

Comment:
Can we mark this as `#[deprecated]`?

Reply:
done

Reply:
this causes serde to warn about the deprecated field. And I wasn't able to fix that will `#[allow(deprecated)]`.